pyproject.toml

```diff
@@ -38,12 +38,13 @@ line-length = 79
 filterwarnings = [
     "ignore::sqlalchemy.exc.SADeprecationWarning",
     "ignore::sqlalchemy.exc.SAWarning",
-    "ignore::DeprecationWarning"
+    "ignore::DeprecationWarning",
+    "ignore::bs4.GuessedAtParserWarning"  # using lxml as default parser
 ]
 markers = [
     "ckan_config: patch configuration used by other fixtures via (key, value) pair.",
     "ckan_pytest: test case that is explicitely was rewriten from `nose` style"
 ]
 
 testpaths = ["ckan", "ckanext"]
-addopts = "--strict-markers  --pdbcls=IPython.terminal.debugger:TerminalPdb"
+addopts = "--strict-markers --pdbcls=IPython.terminal.debugger:TerminalPdb -p no:ckan"
```

Comment:
Can you explain what this does?

Reply:
`-p` flag controls pytest plugins that are enabled during the test execution. `-p xxx` means "enable pytest-xxx", `-p no:xxx` means "disable pytest-xxx". If there is no such plugin `-p no:xxx` just silently ignored.

As all the new extension has `pytest-ckan` added to the `dev-requirements.txt`, as long as you have a custom extension installed, you'll get warnings every time you run CKAN core tests. `pytest-ckan` just duplicates original CKAN fixtures and this causes those warnings.

The flag above disables `pytest-ckan` plugin(which is not used by CKAN itself, anyway) if it's present and thus removes noisy warnings

Reply:
This makes sense thanks